todo_app/data/trello_items.py

```diff
@@ -159,40 +185,40 @@ def get_item(id):
     return next((item for item in items if item['id'] == str(id)), None)
 
 
-def add_item(title):
+def add_item(title: str, description: str, idList: str) -> dict:
     """
-    Adds a new item with the specified title to the session.
+    Adds a new card with the specified title and description to the Trello board.
 
     Args:
         title: The title of the item.
+        description: The task description.
+        listName: The id of the list you want to add the card to
 
     Returns:
         item: The saved item.
     """
-    items = get_items()
 
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
+    title = title.replace(" ", "%20")
+    description = description.replace(" ", "%20")
```

Comment:
Nice find here, but there are plenty of characters this won't catch (like a `+` for example) - if we need to do this I'd always recommend taking advantage of others' hard work and using a library (e.g. [see this SO discussion](https://stackoverflow.com/questions/5607551/how-to-urlencode-a-querystring-in-python)) but here we could actually avoid the issue by passing the data through in the body of the POST request rather than as query params - 

```python
    body = {
        'name': title,
        'desc': description
    }

...

response = requests.post(apiCall.url, params=apiCall.requestAuthPayload, data=body)
```